specifyweb/frontend/js_src/lib/components/DataModel/types.ts

```diff
@@ -6556,3 +6559,237 @@ export type CollectionObjectGroupType = {
   readonly toManyDependent: RR<never, never>;
   readonly toManyIndependent: RR<never, never>;
 };
+export type AbsoluteAge = {
+  readonly tableName: 'AbsoluteAge';
+  readonly fields: {
+    readonly absoluteAge: number | null;
+    readonly ageType: string | null;
+    readonly ageUncertainty: number | null;
+    readonly collectionDate: string | null;
+    readonly date1: string | null;
+    readonly date2: string | null;
+    readonly datingMethod: string | null;
+    readonly datingMethodRemarks: string | null;
+    readonly number1: number | null;
+    readonly number2: number | null;
+    readonly remarks: string | null;
+    readonly text1: string | null;
+    readonly text2: string | null;
+    readonly yesno1: boolean | null;
+    readonly yesno2: boolean | null;
+    readonly timestampCreated: string;
+    readonly timestampModified: string | null;
+  };
+  readonly toOneDependent: RR<never, never>;
+  readonly toOneIndependent: {
+    readonly ageAttachment: AbsoluteAgeAttachment | null;
+    readonly agent1: Agent | null;
+    readonly collectionObject: CollectionObject;
+    readonly createdByAgent: Agent | null;
+    readonly modifiedByAgent: Agent | null;
+  };
+  readonly toManyDependent: RR<never, never>;
+  readonly toManyIndependent: RR<never, never>;
+};
+export type RelativeAge = {
+  readonly tableName: 'RelativeAge';
+  readonly fields: {
+    readonly ageRype: string | null;
+    readonly ageUncertainty: number | null;
+    readonly collectionDate: string | null;
+    readonly date1: string | null;
+    readonly date2: string | null;
+    readonly datingMethod: string | null;
+    readonly datingMethodRemarks: string | null;
+    readonly relativeAgePeriod: number | null;
+    readonly number1: number | null;
+    readonly number2: number | null;
+    readonly remarks: string | null;
+    readonly text1: string | null;
+    readonly text2: string | null;
+    readonly yesno1: boolean | null;
+    readonly yesno2: boolean | null;
+    readonly timestampCreated: string;
+    readonly timestampModified: string | null;
+    readonly verbatimName: string | null;
+    readonly verbatimPeriod: string | null;
+  };
+  readonly toOneDependent: RR<never, never>;
+  readonly toOneIndependent: {
+    readonly ageAttachment: RelativeAgeAttachment | null;
+    readonly ageName: GeologicTimePeriod | null;
```

Comment:
Based on our discussion with the Swiss and GAC, we need an `ageNameEnd` field as well.